packages/desktop-client/src/components/FinancesApp.js

```diff
@@ -57,118 +56,82 @@ import PostsOfflineNotification from './schedules/PostsOfflineNotification';
 import Settings from './settings';
 import Titlebar, { TitlebarProvider } from './Titlebar';
 
-function PageRoute({
-  path,
-  component: Component,
-  redirectTo = '/budget',
-  worksInNarrow = true,
-}) {
+function NarrowNotSupported({ children, redirectTo = '/budget' }) {
```

Comment:
Open to feedback on this `NarrowNotSupported` component as a tidier way to redirect in the few spots it's needed. 

I do think redirecting is important, so folks can safely switch sizes without landing on an unusable page. For example, going from full screen to split view on a tablet or rotating a phone landscape to portrait.